helpers/combine_docfiles.sh

```diff
@@ -0,0 +1,133 @@
+#! /bin/bash
```

Comment:
I think moving this into Bash from Python reduces ease of updates and shouldn't be done lightly.

Reply:
Cool, I'll look at cleaning up the python.  I'm sure there's a fix, just need to figure out a way to run the search and replace without deleting the trailing newline.

FWIW, I personally tend to prefer bash for build / check scripts in FOSS projects for the same reason of ease of updates.  I've found most contributors know enough bash to contribute fixes whereas a smaller subset of contributors are comfortable enough to contribute in other languages.  That is to say bash is sorta like the english language of foss projects.  Sorta awkward, but broadly spoken and understood.

Reply:
@morgante I've removed `combine_docfiles.sh` and improved the existing `combine_docfiles.py` to:

1. Use raw strings for regular expressions to avoid [W608](https://lintlyci.github.io/Flake8Rules/rules/W605.html) pyflake warnings.
2. Simple fix for newline: `output = ... + '\n'`